File: src/population/IPopulationView.cs
using System;
using System.Collections.Generic;
using System.Linq;
using NetTopologySuite.Geometries;
using NetTopologySuite.Index.KdTree;
using NetTopologySuite.Algorithm.Locate;

namespace DVAN.Population
{
    public interface IPopulationView
    {
        public Coordinate getCoordinate(int index);

        public Coordinate getCoordinate(int index, string crs);

        public int getPopulation(int index);

        public int pointCount();

        public List<int> getPointsInEnvelop(Envelope? envelope);
    }
}

File: src/main/java/org/tud/oas/population/IPopulationView.java
package org.tud.oas.population;

import java.util.List;

import org.locationtech.jts.geom.Coordinate;
import org.locationtech.jts.geom.Envelope;

public interface IPopulationView {
    public Coordinate getCoordinate(int index);

    public Coordinate getCoordinate(int index, String crs);

    public int getPopulation(int index);

    public int pointCount();

    public List<Integer> getPointsInEnvelop(Envelope envelope);
}